addons/docs/src/frameworks/vue3/prepareForInline.ts

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import * as Vue from 'vue';
 import { StoryFn, StoryContext } from '@storybook/addons';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
![image](https://user-images.githubusercontent.com/3842800/128085039-1cf5a63c-707a-427e-8579-aa0f02b49990.png)

Reply:
I'm not sure why eslint is complaining now 🤔  `@storybook/vue3` is listed as peerDep

Reply:
It isn't, it was removed in https://github.com/storybookjs/storybook/pull/15413/commits/aff8f1905be34ea04262ab9183f5d9158e9618e0

Reply:
🤦🏻